ChecksFile.wl

```diff
@@ -163,7 +163,7 @@ Return[{\[CapitalOmega]p,kp,\[Omega]}];
 IdisBottom[data_,\[CapitalOmega]_,k_,En_,xflavor_]:=Module[{cos\[Theta],\[Phi]0,\[Phi]1,\[CapitalOmega]p,kp,\[Omega],mu,mubar,Vmatter,\[CapitalOmega]minuskpcos\[Theta],result,bottom},
 cos\[Theta]=data["mids"];
 {\[CapitalOmega]p,kp,\[Omega]}=IdisShifts[data,\[CapitalOmega],k,En,xflavor];
-bottom=Reap[Do[Sow[\[CapitalOmega]p-kp cos\[Theta][[i]]],{i,1,Length[cos\[Theta]]}]][[2,1]];
+bottom=\[CapitalOmega]p-kp cos\[Theta];
```

Comment:
Here it looks like bottom is jsut a new name

Reply:
Oh, it's not changing the name of anything. Right now you fill out the components of the vector with a loop. But the loop is unnecessary, since it is implied in just regular multiplication. scalar*vector=vector.